docs/CONTRIBUTING.internal.md

```diff
@@ -0,0 +1,34 @@
+# How we use GitHub Issues
+
+> Note: this page is intended for Octopus team members only. If you're an Octopus user and you've found a bug or something isn't working, please get in touch with [our support team](https://octopus.com/support) instead.
+
+We use this public GitHub repository as a way to communicate with our community about what we are working on for the core Octopus product.
+
+Different teams across Octopus use the public issues in this repository for the following purposes:
+
+- **Engineers**: Use public issues for linking Pull Requests only. This helps with automatically release note generation.
+- **Support** and **Customer Success** teams:
+  - If you want someone to address an issue, you still need to bring it to the attention of the appropriate team internally, for example, via the requests channel of a Quick Reaction Force.
+  - Use public issues to aid communication with customers while waiting on a possible fix. Customers can subscribe to these GitHub issues, you can use them to collate and track relevant support tickets, and publish workarounds for multiple customers in a single place.
+- **Product** teams: Product teams own the Issues list, they will close anything that's unlikely to be fixed and provide context to Support and Customer Success teams as necessary to help manage expectations.
+
+## Which repository should I create the issue in?
+
+If you want to track an Issue that doesn't deserve public attention, track it with an Issue in the [Private GitHub repository](https://github.com/OctopusDeploy/OctopusDeploy/issues). This way we won't lose track of the item, and it won't be involved in automatic release note generation.
```

Comment:
> and it won't be involved in automatic release note generation
I'm pretty sure we can read release notes from the private repo.

Reply:
Thanks. Removed that comment.